SBJson.xcodeproj/project.pbxproj

```diff
@@ -630,9 +708,11 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 39;
 				FRAMEWORK_VERSION = A;
+				GCC_OPTIMIZATION_LEVEL = 0;
```

Comment:
This setting is only in the Debug configuration. Otherwise it's hard to debug if you consume the framework via Carthage.
Release is still default `Fastest, Smallest [-Os]`.